e2e/tests/adminManagesLocalAuthorities_test.js

```diff
@@ -219,7 +214,9 @@ Scenario('HMCTS admin transfer case to secondary local authority @flaky', async
 
   manageLocalAuthoritiesEventPage.selectTransferLocalAuthority();
   await I.goToNextPage();
```

Comment:
Do we mean to do goToNextPage() followed by goToPreviousPage()?

Reply:
The LocalAuthority name is not shown properly at the first time. But it can be shown if user go back to the previous page and renter to this page. 
The FPL code looks fine to me, not sure it is XUI bug or not.